test/integration/happy-path.test.js

```diff
@@ -0,0 +1,30 @@
+import healthcheck from '../../src/main';
+import { getFixtureDetails, getSpyResults } from '../support/fixtureHelpers';
+
+test('Integration - Happy path, everything 200', async () => {
+  const spy = jest.spyOn(console, 'log');
+  
+  const { config, route } = getFixtureDetails('happy-path');
+  const opts = Object.assign({}, config, {
+    uri: route
+  });
+
+  try {
+    await healthcheck.run(opts);
+  } catch(e) {}
+
+  expect(getSpyResults(spy)).toRoughlyMatchArray([
+    'ℹ️   Requesting http://localhost:5555/fixtures/happy-path',
+    'ℹ️   Asset regex',
+    'ℹ️    - /(main.js|style.css)$/g',
+    '✅   Document loaded OK',
+    '✅   200: http://localhost:5555/fixtures/happy-path/assets/style.css',
```

Comment:
The test runner sets port to  `process.env.PORT`. This would fail if port is not 5555.

Reply:
Good call, have updated to read from config